engine/helpers.go

```diff
@@ -454,13 +458,13 @@ func GetCollatedExchangeAccountInfoByCoin(exchAccounts []exchange.AccountInfo) m
 }
 
 // GetAccountCurrencyInfoByExchangeName returns info for an exchange
-func GetAccountCurrencyInfoByExchangeName(accounts []exchange.AccountInfo, exchangeName string) (exchange.AccountInfo, error) {
-	for i := 0; i < len(accounts); i++ {
+func GetAccountCurrencyInfoByExchangeName(accounts []account.Holdings, exchangeName string) (account.Holdings, error) {
```

Comment:
You've now made a handy helper in `account.GetHoldings()`, so I don't think this func is needed anymore. Plus its only ever been referenced in tests